mage_integrations/mage_integrations/destinations/salesforce/README.md

```diff
@@ -0,0 +1,28 @@
+# Salesforce
+
+![](https://user-images.githubusercontent.com/78053898/198753571-7c033641-75a2-4338-b257-ee37a159915e.png)
+
+<br />
+
+## Config
+
+You must enter the following credentials when configuring this destination:
+
+| Key | Description | Required or optional
+| --- | --- | --- |
+client_id | OAuth client_id | optional|
+client_secret | OAuth client_secret | optional|
+refresh_token | OAuth refresh_token | optional|
+username | User/password username |  optional|
+password  |  User/password password| optional|
+security_token | User/password generated security token. Reset under your Account Settings| optional|
+domain | Your Salesforce instance domain. Use 'login' (default) or 'test' (sandbox), or Salesforce My domain.| Required|
+action | How to handle incomming records by default (insert/update/upsert/delete/hard_delete). Default is "insert"| Required|
+allow_failures | Allows the target to continue persisting if a record fails to commit. Default is "False"| optional| 
+table_name | Allows the target to use a different source to this destination. (Read "limitations" section)| optional|
+
+<br />
+
```

Comment:
You can reference this doc for getting credentials: https://github.com/mage-ai/mage-ai/blob/master/mage_integrations/mage_integrations/sources/salesforce/README.md#how-to-get-your-client_id-client_secret-and-refresh_token

Reply:
added on 940f3032c47cd0d717a20e0b1e94baf3bcd218a5